packages/react-native/src/message-list/message-list.style.ts

```diff
@@ -19,6 +23,10 @@ const darkColors = {
   unreadColor: "#2a2a39",
   reactionBorder: "#28293d",
   reactionActiveBackground: "rgba(239, 58, 67, 0.3)",
+  downloadedSuccessBannerBackground: "rgba(224, 224, 224, 0.9)",
```

Comment:
I think the colors here should be the other way around. Currently the modal has white background on dark mode and almost black on light mode